.circleci/config.yml

```diff
@@ -601,16 +608,6 @@ jobs:
           path: *TEST_RESULTS_DIR
       - run: *notify-slack-failure
 
-  # run integration tests on nomad/main
-  nomad-integration-main:
-    docker:
-      - image: docker.mirror.hashicorp.services/cimg/go:1.18
-    environment:
-      <<: *ENVIRONMENT
-      NOMAD_WORKING_DIR: /home/circleci/go/src/github.com/hashicorp/nomad
-      NOMAD_VERSION: main
-    steps: *NOMAD_INTEGRATION_TEST_STEPS
-
```

Comment:
I'm a little cautious about creating nightly jobs unless they are very loud when they fail, otherwise, they will rot pretty readily. If I'm not mistaken though, this integration test exists in the first place because there was some regression that happened at some point. I guess I would leave this for more follow-up work. I want to do more of this async testing at some point and this would fit into that story. I would imagine something like what the K8s team has for their acceptance tests where it runs async but notifies their Slack channel so that it's hard to miss.